src/components/createPDF.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react'
-import { Button, Box } from "@material-ui/core"
+import { Button, Box } from '@material-ui/core'
 import { createPDFPopup } from '../pdfFiller/fill1040Fields'
-import { Link } from "react-router-dom"
+import { Link } from 'react-router-dom'
 
-export default function CreatePDF() {
-    return (
+export default function CreatePDF () {
+  return (
         <Box display="flex" justifyContent="center">
             <form>
-                <Box display="flex" justifyContent="flex-start">   
+                <Box display="flex" justifyContent="flex-start">
                     <h2>Print Copy to File</h2>
                 </Box>
 
                 <Box display="flex" justifyContent="flex-start" paddingTop={2} paddingBottom={1}>
                     <Box display="flex" justifyContent="flex-start" paddingRight={2}>
-                        <Button component={Link} to={"familyinfo"} variant="contained" color="secondary" >
+                        <Button component={Link} to={'familyinfo'} variant="contained" color="secondary" >
```

Comment:
There are a number of these where a plain string is inside jsx `{}`. Is there a reason?

Reply:
That's a good callout. I didn't realize that the link strings could be added without the curly braces but that totally makes sense given that it's static rather than a variable.

Reply:
@thegrims I added another commit to remove all those plain strings in {}s

Reply:
Awesome, thanks!